src/Microsoft.OData.Core/Json/TextWriterWrapper.cs

```diff
@@ -76,6 +77,30 @@ public override void Flush()
             this.writer.Flush();
         }
 
+        /// <summary>
+        /// Increases the level of indentation applied to the output asynchronously.
+        /// </summary>
+        public virtual async Task IncreaseIndentationAsync()
+        {
+            await TaskUtils.CompletedTask.ConfigureAwait(false);

```

Comment:
>TaskUtils.CompletedTask [](start = 18, length = 23)

In 8.0, hopefully we can move to Task.CompletedTask/Task.FromResult instead of TaskUtils...